Makefile

```diff
@@ -140,7 +140,7 @@ lint:
 	woke
 	golangci-lint run
 	find . -type f -path './**/*.*sh' -not -path '*vendor*' | xargs -r shellcheck
-	operator-sdk bundle validate ./olm-catalog/serverless-operator
+	operator-sdk bundle validate ./olm-catalog/serverless-operator --select-optional suite=operatorframework --optional-values=k8s-version=1.22
```

Comment:
just curious, why set we the optional value for `k8s-version` to `1.22` ?

Reply:
To check stuff against OCP 4.9 (which is k8s 1.22)

Reply:
but we have 4.7 or 4.8 as well as test runs - thats why I was wondering on a specific version for the CLI argument.

Is it to ensure to validate to higher than 1.22? :thinking:

Reply:
Nope, it's a straight copy from main where we don't run 4.9 presubmit.

Reply:
OK. I guess I don't understand this, why the 1.22 was added to check against 4.9 :smile: :man_shrugging: 
(e.g. instead of 1.21)

Reply:
ah... this checks the compatibility (as in APIs etc) for the 1.22 version... Now I get it 

Thanks for infos!